corehq/apps/locations/migrations/0006_auto_20161114_1858.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.11 on 2016-11-14 18:58
+from __future__ import unicode_literals
+
+from django.db import migrations
+import django.db.models.manager
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('locations', '0005_locationtype_include_without_expanding'),
+    ]
+
+    operations = [
+        migrations.AlterModelManagers(
+            name='sqllocation',
+            managers=[
+                ('_tree_manager', django.db.models.manager.Manager()),
```

Comment:
Is this autogenerated?  This isn't the model manager we use - I think it should be `corehq.apps.locations.models.LocationManager`.  Then again, I have no idea how this is related to a migration, so maybe a base-class is sufficient?